pyccl/tests/test_bcm.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 import numpy as np
 import pyccl as ccl
+from pyccl.pyutils import assert_warns
```

Comment:
FYI: We have an `assert_warns` function in `pyutils` which we use in tests, however I will be removing all that because we test using `pytest` only and `pytest` has the wonderful `with pytest.warns(Warning):` context manager, used in the same way as `pytest.raises`. There is no need to mix in `assert_warns` and `numpy.testing` like we have in some tests, so just keep that it mind next time you add a test for warnings.

`warnings` and `numpy` are good packages, but for other kinds of stuff. We even say we use `pytest` [here](https://ccl.readthedocs.io/en/latest/source/writing_and_running_unit_tests.html) in the docs.

Reply:
Done